app/Helpers/SecurePasswordHasher.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Security.Cryptography;
+
+namespace backend.Helpers
+{
+    // Source: https://stackoverflow.com/questions/4181198/how-to-hash-a-password
+    public static class SecurePasswordHasher
```

Comment:
ok, jestem w stanie się zgodzić, żeby to był static. Czy moglibyśmy to przerobić na singletona? => wszystkie metody niestatyczne, cała klasa niestatyczna, jedno pole prywatne 'instance' i do tego odpowiednia statyczna właściwość Instance, która przy get sprawdzi czy instance nie jest nullem i jeśli jest to go zainicjuje

Reply:
To samo co wyżej czemu chciałbyś, żeby to był singleton? Ta klasa nie przechowuje żadnych danych tylko wykonuje jakieś funkcje czemu to nie może być static i ma być trzymana w pamięci jej instancja (jako singleton)?